test/test_fork.py

```diff
@@ -32,6 +33,10 @@ def setUpModule():
 @skipIf(
     not hasattr(os, "register_at_fork"), "register_at_fork not available in this version of Python"
 )
+@skipIf(
+    "gevent" in sys.modules.keys() or "eventlet" in sys.modules.keys(),
```

Comment:
You can use `in sys.modules`, `.keys()` is not needed.

Reply:
Let's use is_greenthread_patched from test/utils.py instead.